engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -254,6 +254,7 @@ private void checkOpenGL() {
                 GLContext.getCapabilities().OpenGL12,
                 GLContext.getCapabilities().OpenGL14,
                 GLContext.getCapabilities().OpenGL15,
+                GLContext.getCapabilities().OpenGL20,
```

Comment:
Turns out opengl 2.0 is in use in the code, but was not part of the opengl check.